app/views/DefaultBrowserView/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useEffect, useLayoutEffect, useState } from 'react';
+import { FlatList, Linking } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+import I18n from '../../i18n';
+import { useTheme } from '../../theme';
+import StatusBar from '../../containers/StatusBar';
+import * as List from '../../containers/List';
+import { DEFAULT_BROWSER_KEY } from '../../lib/methods/helpers/openLink';
+import { isIOS } from '../../lib/methods/helpers';
+import SafeAreaView from '../../containers/SafeAreaView';
+import UserPreferences from '../../lib/methods/userPreferences';
+import { events, logEvent } from '../../lib/methods/helpers/log';
+import Item from './Item';
+
+export type TValue = 'inApp' | 'systemDefault:' | 'googlechrome:' | 'firefox:' | 'brave:';
+
+export interface IBrowsersValues {
+	title: string;
+	value: TValue;
+}
+
+const DEFAULT_BROWSERS: IBrowsersValues[] = [
+	{
+		title: 'In_app',
+		value: 'inApp'
+	},
+	{
+		title: isIOS ? 'Safari' : 'Browser',
+		value: 'systemDefault:'
+	}
+];
+
+const BROWSERS: IBrowsersValues[] = [
+	{
+		title: 'Chrome',
+		value: 'googlechrome:'
+	},
+	{
+		title: 'Firefox',
+		value: 'firefox:'
+	},
+	{
+		title: 'Brave',
+		value: 'brave:'
+	}
+];
+
+const DefaultBrowserView = () => {
+	const [browser, setBrowser] = useState<string | null>(null);
+	const [supported, setSupported] = useState<IBrowsersValues[]>([]);
+
+	const { theme } = useTheme();
+	const navigation = useNavigation();
+
+	useLayoutEffect(() => {
+		navigation.setOptions({
+			title: I18n.t('Close_Chat')
+		});
+	}, [navigation]);
+
+	useEffect(() => {
+		const getBrowser = UserPreferences.getString(DEFAULT_BROWSER_KEY);
+		setBrowser(getBrowser);
+
+		if (isIOS) {
```

Comment:
The old code doesn't have this logic, are you sure it's just for ios?

Reply:
Actually, there is.

https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4424/files#diff-4161ff0a346cc7ff7d7339326a01e6ca3a700901f02ef5160730cb90013625a4L69-L71